### PR TITLE
Add new dbnsfp 4.8a

### PR DESCRIPTION
**Pull request by everaldorodrigo**
# Objective: Implement dbNSFP 4.8a
Columns added (dbNSFP_variant):
- PHACTboost_score
- PHACTboost_rankscore
- MutFormer_score
- MutFormer_rankscore
- GERP_91_mammals
- GERP_91_mammals_rankscore

### CODE REVIEW

**Comment by everaldorodrigo** [comment]
# Objective: Implement dbNSFP 4.8a
Columns added (dbNSFP_variant):
- PHACTboost_score
- PHACTboost_rankscore
- MutFormer_score
- MutFormer_rankscore
- GERP_91_mammals
- GERP_91_mammals_rankscore

File: src/hub/dataload/sources/dbnsfp/dbnsfp_mapping_48a_v1.py [hunk 1]
```diff
@@ -0,0 +1,997 @@
...
+                    },
+                    "rs": {
+                        "type": "float"
+                    },
+                    "rs_rankscore": {
+                        "type": "float"
+                    }
+                }
+            },
+            "gerp": {
```
Comment (everaldorodrigo) [inline src/hub/dataload/sources/dbnsfp/dbnsfp_mapping_48a_v1.py:481]: @newgene, do you think the `gerp` field  is well structured in the mapping? It was also added to the mapping v2.

Comment (newgene) [reply src/hub/dataload/sources/dbnsfp/dbnsfp_mapping_48a_v1.py:481]: not sure, have you checked against the actual data? If it matches the data, it should be fine.

Comment (everaldorodrigo) [reply src/hub/dataload/sources/dbnsfp/dbnsfp_mapping_48a_v1.py:481]: Yes I have. It's following the same pattern of other fields.

**Pull request merged**